docs/eventlisteners.md

```diff
@@ -307,6 +307,31 @@ can be used where we only provide single argument for `matchNames` as `*`.
     - *
 ```
 
+### LabelSelector
+This field determines the labels for which EventListener will select triggers and
+create Tekton resources.
+
+Snippet below will function the eventlistener namespace and only select triggers with 
```

Comment:
```suggestion
In the snippet below, the EventListener will only select triggers from its's  namespace with 
```

Reply:
"its namespace" - this is a possessive, so no apostrophe ("it's" is short for "it is").

Reply:
yeah, updated to `its namespace`